components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +428,17 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        if (reCaptchaScoreThreshold == null) {
```

Comment:
Let's use stringUtils.isNotBlank

Reply:
Used stringUtils.isBlank

Reply:
why did you remove stringUtils.isBlank check?

Reply:
Sorry. my mistake